README.md

```diff
@@ -13,19 +13,30 @@ In 2019 we create [Undebates](https://github.com/EnCiv/undebate) and launched th
 
 # Getting Started
 
-The following has been tested using node v16.13.0 and npm v8.1.0 on Windows 10 and MacOs.  There are know issues using nmp < 7. During install, you can ignore the warning about incompatibility with previous previous versions of node/npm - we need to go back to those packages and update their dependencies.
+The following has been tested using node v16.13.0 and npm v8.1.0 on Windows 10 and MacOs.  There are known issues using npm < 7. During install, you can ignore the warning about incompatibility with previous versions of node/npm - we need to go back to those packages and update their dependencies.
```

Comment:
Since you are editing this please add something like this:
Node version 17 will not work, and in general the odd versions are experimental so we don't try to use those. If you already tried to build this with version 17, you need to install version 16 and then do `npm ci` to start clean.

Reply:
Add your comment about Node v17.